### DISCUSSION TITLE
Support for "secure keys", e.g. Yubikey

### DISCUSSION THREAD

**Discussion by antonmos** [zoidsh/meru#272] [Feature Requests]
Currently, we get "This browser does not support security keys" messages with a link to install Chrome.

![image](https://user-images.githubusercontent.com/2713/114420471-f80ece00-9b79-11eb-9ac2-6f87e7e0c85e.png)

**Comment by timche** [comment]
Electron doesn't support WebAuthn yet. There is an active PR in progress though: https://github.com/electron/electron/pull/28349 So we need to wait until Electron supports it.

**Reply by antonmos** [reply]
thank you for the quick response!
